pkg/blobs/local_storage.go

```diff
@@ -37,11 +37,18 @@ func newLocalStorage(externalIODir string) (*localStorage, error) {
 	return &localStorage{externalIODir: absPath}, nil
 }
 
+// prependExternalIODir makes `path` relative to the configured external I/O directory.
+//
+// Note that we purposefully only rely on the simplified cleanup
+// performed by filepath.Join() - which is limited to stripping out
+// occurrences of "../" - because we intendedly want to allow
+// operators to "open up" their I/O directory via symlinks. Therefore,
+// a full check via filepath.Abs() would be inadequate.
 func (l *localStorage) prependExternalIODir(path string) (string, error) {
 	localBase := filepath.Join(l.externalIODir, path)
-	// Make sure we didn't ../ our way back out.
+	// Make sure no symlinks accessed a way out.
```

Comment:
isn't this comment the opposite of what the code / comment above is doing?

Reply:
woops. Corrected during the rebase.